user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/storage_per_DMA/Profile_migration_to_elastic.md

```diff
@@ -0,0 +1,96 @@
+---
+uid: Profile_migration_to_elastic
+---
+
+# Migrating SRM profiles to the indexing database
+
+Starting from DataMiner version 10.1.0/10.0.8<!-- RN 25758 -->, you can migrate the profiles from the *Profiles.xml* file to the indexing database. This improves the scalability and performance on systems that have a large number of profiles.
+
+When you [install an indexing database](xref:Installing_Elasticsearch_via_DataMiner) and [migrate booking data](xref:Configuring_DataMiner_Indexing) on an existing DataMiner System prior to DataMiner 10.4.0, profiles are not automatically migrated. Based on your specific setup, and keeping in mind the [limitations and differences with XML storage](#limitations-and-differences-with-xml-storage), you can decide independently whether or not you want to start this migration. However, **from DataMiner 10.4.0 onwards**, the use of an indexing database (or Storage as a Service) to store profiles is **mandatory**. Using XML storage is no longer possible from that DataMiner version onwards.
+
+> [!TIP]
+> If you are looking for information on how to migrate an SRM configuration from one DMA to another, see [Migrating an SRM configuration](xref:SRM_migrating).
+
+> [!NOTE]
+>
+> - XML storage is no longer supported from DataMiner 10.4.0 onwards. Upgrading to 10.4.0 will not be possible if the profile data is still stored in XML.
+> - Elasticsearch is only supported up to version 6.8. If you are using Elasticsearch as the indexing database, we therefore recommend using [Storage as a Service](xref:STaaS) instead.
+
+## Migrating from XML to the indexing database
+
+To migrate the profiles, you will need to use the SLNetClientTest tool.
```

Comment:
On the page about resources migration, it says that this is also supported for an OpenSearch database. Is OpenSearch not supported for the profiles migration?

Reply:
Based off documentation [Software_support_life_cycles](https://docs.dataminer.services/user-guide/Reference/Software_support_life_cycles.html#third-party-software-support-life-cycle), we didn't reference OpenSearch

Reply:
Possibly this is based on a misinterpretation then, because that page states "Note that using a self-hosted OpenSearch database remains supported." We continue to support OpenSearch as an alternative for users who do not want to use STaaS.